memtagabielf64/LICENSE

```diff
@@ -1,22 +1,202 @@
-This work is licensed under the Creative Commons
```

Comment:
Thank you for responding to the review feedback regarding copyright statements, but please do not change the LICENSE file.
The contribution you're making is still under the same CC with patent grant license used in the other parts of the ABI specifications in this project; not under the Apache2.0 license as the proposed change to this LICENSE file would imply.
The only thing that you need to change is to add a line to the copyright header - I'll point to the right place to do so in another comment.